packages/polymath-issuer/src/actions/sto.js

```diff
@@ -308,6 +308,23 @@ export const configureSTO = (config: STOConfig, type: string) => async (
   );
 };
 
+export const removeTier = (id, tiers, setFieldValue) => async (
+  dispatch: Function,
+  getState: GetState
+) => {
+  dispatch(
+    ui.confirm(
+      <div>
+        <p>Please confirm that you want to delete this tier</p>
+      </div>,
+      async () => {
+        tiers.splice(id - 1, 1);
```

Comment:
Lets clone this array before splicing to avoid mutating the reference of `tiers`

```js
const newTiers = { ...tiers };
newTiers.splice(id - 1, 1);
// ... etc
```

Reply:
Resolved with latest commit